mne/viz/evoked.py

```diff
@@ -1310,7 +1310,7 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
 
     # butterfly/time series plot
     # most of this code is about passing defaults on demand
-    ts_ax = fig.add_subplot(212)
+    ts_ax = fig.add_subplot(212, label='timeaxes')
```

Comment:
@jona-sassenhagen this is to take care of this warning:
```
mne/viz/tests/test_topo.py::test_plot_topo
  /home/larsoner/custombuilds/matplotlib/lib/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
```
Does it make sense to you?

Reply:
Actually .. why is that line there at all? The tests I tried pass when it's deleted, plots look fine too, and it seems to be overwriting the identical axes coming from `_prepare_joint_axes` above.